portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,44 @@
+/* The main body */
+body {
+  font-family: sans-serif
+}
+
+/* The sidebar */
+.sidebar {
+  background-color: #bfbfbf;
+  left: 10px;
+  overflow-x: hidden;
+  padding-top: 20px;
+  position: fixed;
+  top: 10px;
+  z-index: 1;
+}
+
+/* Links on the sidebar */
+.sidebar a {
+  color: #000000;
+  display: block;
+  font-size: 25px;
+  padding: 2px 20px 20px 20px;
```

Comment:
This can be simplified to:
padding: 2px 20px 20px;

https://www.w3schools.com/css/css_padding.asp - particularly around Shorthand

Reply:
Thanks! Fixed pushed.